rpcs3/Emu/Cell/Modules/sceNp2.cpp

```diff
@@ -423,11 +423,12 @@ error_code sceNpMatching2SignalingGetConnectionStatus(
 		return SCE_NP_MATCHING2_ERROR_NOT_INITIALIZED;
 	}
 
-	const auto& infos = nph->get_peer_infos(ctxId, roomId, memberId);
+	const auto sigh = g_fxo->get<named_thread<signaling_handler>>();
+	const auto si = sigh->get_sig2_infos(roomId, memberId);
 
-	*connStatus           = infos.connStatus;
-	(*peerAddr).np_s_addr = infos.addr; // infos.addr is already BE
-	*peerPort             = std::bit_cast<u16, be_t<u16>>(infos.port); // infos.port is already BE
+	*connStatus           = si.connStatus;
+	(*peerAddr).np_s_addr = si.addr; // infos.addr is already BE
+	*peerPort             = si.port;
```

Comment:
connStatus, peerAddr, peerPort deference without test? (or is my github view bust)